build.sbt

```diff
@@ -2,6 +2,18 @@ val ioeffect = project.settings(
   name := "scalaz-ioeffect"
 )
 
+val `ioeffect-cats` = project
```

Comment:
I'd have just called it `cats`, less typing in the shell...

Reply:
I figured that the publishing options would be derived from the project name thus would be published under `ioeffect-cats` by default.

I know you pointed this out on gitter but that's why I made it that way. pulling in `"..scalaz" %% "cats % "0.blah"` is streange

Reply:
the publishing name is derived from `name := ` which you've set already.